sdk/tools/spec2def/spec2def.c

```diff
@@ -1365,7 +1400,70 @@ void usage(void)
            "  --implib                generate a def file for an import library\n"
            "  --no-private-warnings   suppress warnings about symbols that should be -private\n"
            "  -a=<arch>               set architecture to <arch> (i386, x86_64, arm)\n"
-           "  --with-tracing          generate wine-like \"+relay\" trace trampolines (needs -s)\n");
+           "  --with-tracing          generate wine-like \"+relay\" trace trampolines (needs -s)\n"
+           "  --noroscompat           don't generate ros-compat information\n");
+}
+
+// HACK, because mingw headers
+#define MIN(a,b) ((a) <= (b) ? (a) : (b))
+
+int CompareExports(const void* pLeft,const void* pRight)
+{
+    EXPORT *pexpLeft = (EXPORT*)pLeft;
+    EXPORT *pexpRight = (EXPORT*)pRight;
+    int result = strncmp(pexpLeft->strName.buf,
+                         pexpRight->strName.buf,
+                         MIN(pexpLeft->strName.len,
+                             pexpRight->strName.len));
+    if (result == 0)
+    {
+        return (pexpLeft->strName.len < pexpRight->strName.len) ? -1 : 1;
+    }
+
+    return result;
+}
+
+void
+Output_RosCompatDescriptor(FILE *file, EXPORT *pexports, unsigned int cExports)
```

Comment:
I'd suggest generating the regular export directory, not use the .def file and call the C file <module>_exports.c

That would pave the way for relay tracing :-)